docs/source/pillars/data_management.md

```diff
@@ -149,6 +149,24 @@ The ability of the TRE operator to ensure outputs are safely published and share
     Results will be produced which address the project's purpose, some of which may be outputs that are removed from the TRE.
     Understanding what these outputs are likely to be and their sensitivity as early as possible will help prepare for their processing and publication.
   - Recommended
+* - 3.3.3.
+  - You must have a documented process for disclosure control of output files from the TRE.
```

Comment:
```suggestion
  - You must have a documented process for disclosure control of outputs from the TRE.
```

We should be careful, I don't think we can assume or guarantee that outputs are always files.
I could, for example, imagine a TRE which produces outputs through some private or public API.